CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Unecessary `Cell` rerender on `EXPERIMENTAL_Table`.
```

Comment:
maybe add just only a sentence saying how you did it, could be improve a lot this log.

Reply:
sure!